SYCL/Basic/device_platform_interop.cpp

```diff
@@ -0,0 +1,46 @@
+// REQUIRES: opencl
+
+// RUN: %clangxx -fsycl %s -o %t.run
+// RUN: %GPU_RUN_PLACEHOLDER %t.run
+// RUN: %CPU_RUN_PLACEHOLDER %t.run
+// RUN: %ACC_RUN_PLACEHOLDER %t.run
+
+//==--------device_platform_interop.cpp - SYCL device and platform interop
```

Comment:
line  is too long

Reply:
Do I need to reduce length of comment or move part of it on the next line?

Reply:
Fixed